.travis.yml

```diff
@@ -1,6 +1,25 @@
 language: ruby
 cache: bundler
+sudo: required
+services:
```

Comment:
I would ask the team first. As a developer, I would expect the CI to run the tests at every branch.

Reply:
It would still do so, but only if you open up a pull request. If you haven't opened up a pull request the code is probably not worth testing it.  
Currently the tests are always running twice if you open up a pull request which is unnecessary.